package.json

```diff
@@ -1,202 +1,216 @@
+				"icon": "resources/search.png"
+			},
+			{
+				"command": "leetcode.testSolution",
+				"title": "Test in LeetCode",
+				"category": "LeetCode"
+			},
+			{
+				"command": "leetcode.submitSolution",
+				"title": "Submit to LeetCode",
+				"category": "LeetCode"
+			}
+		],
+		"viewsContainers": {
+			"activitybar": [
+				{
+					"id": "leetcode",
+					"title": "LeetCode",
+					"icon": "resources/LeetCode.svg"
+				}
+			]
+		},
+		"views": {
+			"leetcode": [
+				{
+					"id": "leetCodeExplorer",
+					"name": "LeetCode"
+				}
+			]
+		},
+		"menus": {
+			"view/title": [
+				{
+					"command": "leetcode.searchProblem",
+					"when": "view == leetCodeExplorer",
+					"group": "navigation@1"
+				},
+				{
+					"command": "leetcode.refreshExplorer",
+					"when": "view == leetCodeExplorer",
+					"group": "navigation@2"
+				}
+			],
+			"view/item/context": [
+				{
+					"command": "leetcode.showProblem",
+					"when": "view == leetCodeExplorer && viewItem == problem",
+					"group": "leetcode-explorer@1"
+				}
+			],
+			"commandPalette": [
+				{
+					"command": "leetcode.showProblem",
+					"when": "never"
+				}
+			],
+			"explorer/context": [
+				{
+					"command": "leetcode.testSolution",
+					"when": "explorerResourceIsFolder == false",
+					"group": "file-explorer@1"
+				},
+				{
+					"command": "leetcode.submitSolution",
+					"when": "explorerResourceIsFolder == false",
+					"group": "file-explorer@2"
+				}
+			]
+		},
+		"configuration": [
+			{
+				"title": "LeetCode",
+				"properties": {
+					"leetcode.showLocked": {
+						"type": "boolean",
+						"default": false,
+						"scope": "window",
+						"description": "Show locked problems."
+					},
+					"leetcode.defaultLanguage": {
+						"type": "string",
+						"enum": [
+							"bash",
+							"c",
+							"cpp",
+							"csharp",
+							"golang",
+							"java",
+							"javascript",
+							"kotlin",
+							"mysql",
+							"python",
+							"python3",
+							"ruby",
+							"scala",
+							"swift"
+						],
+						"scope": "window",
+						"description": "Default language for solving the problems."
+					},
+					"leetcode.showSetDefaultLanguageHint": {
+						"type": "boolean",
+						"default": true,
+						"scope": "window",
+						"description": "Show a hint to set the default language."
+					}
+				}
+			}
+		]
+	},
+	"scripts": {
+		"vscode:prepublish": "npm run compile",
+		"compile": "tsc -p ./",
+		"watch": "tsc -watch -p ./",
+		"postinstall": "node ./node_modules/vscode/bin/install",
+		"test": "npm run compile && node ./node_modules/vscode/bin/test",
+		"lint": "tslint --project tsconfig.json -e src/*.d.ts -t verbose"
+	},
+	"devDependencies": {
+		"@types/fs-extra": "^5.0.0",
+		"@types/mocha": "^2.2.42",
+		"@types/node": "^7.0.43",
+		"tslint": "^5.9.1",
+		"typescript": "^2.6.1",
+		"vscode": "^1.1.6"
+	},
+	"dependencies": {
+		"fs-extra": "^5.0.0",
+		"leetcode-cli": "2.5.1",
+		"opn": "^5.2.0"
+	},
+	"__metadata": {
```

Comment:
This may not be a problem. Just curious about this field. Do you know where this field comes from?

Reply:
this field could have come from vscode as I modified directly inside program's location on disk ".vscode\extensions" ... I think you can delete it.

Reply:
I see...

Well I'm fine to change this myself, but I encourage that you can make the change directly in this PR, since this awesome feature is originally driven by you.

You can simply push the changes to your branch `ntt2k:master` then the PR here can automatically synced.

The changes we need to do are:
- repleace `tab` to `space`
- remove the `__metadata` fields in `package.jon`

Would you mind to do that? And please feel free to tell me if you have any question. 😄